op-proposer/proposer/service.go

```diff
@@ -203,6 +210,17 @@ func (ps *ProposerService) initL2ooAddress(cfg *CLIConfig) error {
 	return nil
 }
 
+func (ps *ProposerService) initDGF(cfg *CLIConfig) error {
+	l2ooAddress, err := opservice.ParseAddress(cfg.DGFAddress)
+	if err != nil {
+		return nil
+	}
+	ps.DisputeGameFactoryAddr = &l2ooAddress
```

Comment:
I'd rename the variable `l2ooAddress` to smth like `dgfAddress`

Reply:
Good catch, copy pasta